bot.py

```diff
@@ -104,11 +105,13 @@ async def on_message(self, message: Message):
       elif farm.farm_channel != message.channel.id:
         return
       elif farm.last_farmer == message.author.id:
+        await message.add_reaction("❌")
         embed = discord.Embed(
           title="You cannot farm mushrooms now",
           description="You can only farm mushrooms one at a time",
           colour=discord.Colour.red()
         )
+        embed.set_author(name=message.author)
```

Comment:
This isn't correct, and will give an error, if you mean to set the author of the embed to the person who sent the shroom, sure I can do that in the future